docs/EmbeddedSwift/UserManual.md

```diff
@@ -183,7 +183,7 @@ Embedded Swift is a subset of the Swift language, and some features are not avai
 Features that are not available:
 
 - **Not available**: Runtime reflection (`Mirror` APIs).
-- **Not available**: Values of protocol types ("existentials"), e.g. `let a: Hashable = ...`, are not allowed. `Any` and `AnyObject` are also not allowed.
+- **Not available**: Values of protocol types ("existentials"), unless the protocol is restricted to be class-bound (derived from AnyObject). E.g. `let a: Hashable = ...` is not allowed. `Any` is also not allowed.
```

Comment:
Is the restriction really on the _protocol_, or just the existential type? That is, do we allow `any Hashable & AnyObject`, `any Hashable & SomeClassBoundProtocol`, or `any Hashable & SomeClass`?

Reply:
It is really on the protocol - at least currently.

Reply:
Why?